flake.nix

```diff
@@ -97,6 +99,19 @@
           '';
         };
 
+        devShells.openssl3 = devShells.default.overrideAttrs
+          (finalAttrs: previousAttrs: {
+            # Re-include cmake to update the environment with a new libcrypto.
+            buildInputs = [ pkgs.cmake openssl_3_0 ];
```

Comment:
what happened to tls_packages?

Reply:
Not needed, it turns out the way we're building them includes them in the mkShell.